### ISSUE TITLE
JSON, wrong number of arguments (given 0, expected 1+)

### ISSUE THREAD

**Issue by carlhoerberg** [librato/librato-metrics#122]
```
#<ArgumentError: wrong number of arguments (given 0, expected 1+)> 
  /app/vendor/ruby-2.3.1/lib/ruby/2.3.0/json/common.rb:467:in `JSON' 
  /app/vendor/bundle/ruby/2.3.0/gems/librato-metrics-2.0.0/lib/librato/metrics/persistence/direct.rb:21:in `block in persist' 
  /app/vendor/bundle/ruby/2.3.0/gems/librato-metrics-2.0.0/lib/librato/metrics/persistence/direct.rb:20:in `each' 
  /app/vendor/bundle/ruby/2.3.0/gems/librato-metrics-2.0.0/lib/librato/metrics/persistence/direct.rb:20:in `persist' 
  /app/vendor/bundle/ruby/2.3.0/gems/librato-metrics-2.0.0/lib/librato/metrics/processor.rb:34:in `submit'
```

started happening when upgrading to 2.0.0

**Comment by nextmat** [comment]
Thanks for the bug report, @chancefeick assume you're taking a look at this?

**Comment by chancefeick** [comment]
Patched in 2.0.1 (#123). Thanks for the report, we've added 2.3.1 to our build matrix as a result.
